docs/using-guidelines.asciidoc

```diff
@@ -14,20 +28,20 @@ practices.
 * Map as many fields as possible to ECS.
 
 [float]
-==== Guidelines for writing fields
+===== Guidelines for writing fields
```

Comment:
Why a distinction between "writing fields" and "naming fields"?  Would this be simpler in one list?

Reply:
Just pushed out a rewrite of this section.